examples/react-es6/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "react": "0.13.0-beta.1",
+    "react-tools": "0.13.0-beta.1"
+  },
+  "devDependencies": {
+    "babel-jest": "*"
```

Comment:
Ah yea, you're right. I forgot that `import` and `export` didn't get transformed. Sorry about that. I guess just leave it with using babel.